### PR TITLE
Add more scenarios to new workflow

### PR DESCRIPTION
**Pull request by a-tylenda**
This pull request is related to PR [#183](https://github.com/scramjetorg/transform-hub/pull/183), where a new workflow for npm packages (@scramjet/sth & @scramjet/cli) was created. In this PR the number of tested scenarios was extended and adjusted accordingly.

### CODE REVIEW

**Review by MichalCz — APPROVED** [review]

File: bdd/step-definitions/e2e/cli.ts [hunk 1]
```diff
@@ -7,10 +7,16 @@ import * as fs from "fs";
...
 
-const si = process.env.SCRAMJET_SPAWN_TS
-    ? ["npx", "ts-node", "../packages/cli/src/bin/index.ts"]
-    : ["node", "../dist/cli/bin"]
+// const si = process.env.SCRAMJET_SPAWN_TS
+//     ? ["npx", "ts-node", "../packages/cli/src/bin/index.ts"]
+//     : ["node", "../dist/cli/bin"]
+// ;
+
+const si = process.env.SCRAMJET_SPAWN_JS
```
Comment (alicja-gruzdz) [inline bdd/step-definitions/e2e/cli.ts:15]: I'd add it to our ENV Bible: https://github.com/scramjetorg/transform-hub/blob/release/0.13/ENV_VARS.md :)
so the way I understand it, our tests which are not run with `SCRAMJET_SPAWN_JS` will be testing the CLI installed from npm (the one installed locally)?
Hmm maybe we could mention it somewhere in the tests. I just feel I will find myself debugging the new fixes in CLI and not understanding why they still don't work :smile:

**Review by alicja-gruzdz — APPROVED** [review]

File: bdd/step-definitions/e2e/cli.ts (hunk 1 above)
Comment (alicja-gruzdz) [reply bdd/step-definitions/e2e/cli.ts:15]: as an alternative perhaps allowing one of the three: ts/js/npm?

Comment (a-tylenda) [reply bdd/step-definitions/e2e/cli.ts:11]: new ENV added to our Bible 😉 I also added short description which says that when `SCRAMJET_SPAWN_JS` is set to `false` cli commands are executed from npm package with `si` command.

Comment (alicja-gruzdz) [reply bdd/step-definitions/e2e/cli.ts:11]: excellent, thanks :)

**Pull request merged**